### PR TITLE
Setting version for automatically updating documentation

### PR DESCRIPTION
**Pull request by gwaybio**

### CODE REVIEW

**Comment by gwaybio** [comment]
currently `--editable .` in `requirements.txt` is enabling `import cytominer-database` in the tests, but prevents a sphinx import in `setup.py`.

**Comment by gwaybio** [comment]
Moving autobuild logic to `setup.cfg` file fixes the `setup.py` import bug (see 4a1253194c6b5fecd09279303b438b54273f86ac)

File: setup.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import setuptools
 
+name = "cytominer_database"
```
Comment (shntnu) [inline setup.py]: 💯

File: setup.py [hunk 2]
```diff
@@ -13,31 +15,24 @@
...
-        'click>=6.7',
-        'configparser>=3.5.0',
-        'csvkit>=1.0.2',
-        'pandas>=0.20.3'
+        "backports.tempfile>=1.0rc1",
+        "click>=6.7",
+        "configparser>=3.5.0",
+        "csvkit>=1.0.2",
+        "pandas>=0.20.3",
+        "sphinx>=2.2.2",
```
Comment (shntnu) [inline setup.py]: @gwaygenomics we don't need `sphinx` and `sphinx-rtd-theme` for install, correct?

File: requirements.txt [hunk 3]
```diff
@@ -5,3 +5,8 @@
 pytest>=3.2.2
 sphinx>=1.6.4
 sphinx_rtd_theme>=0.2.5b1
+backports.tempfile>=1.0rc1
```
Comment (shntnu) [inline requirements.txt]: I was able to install, test, and also build the docs even without these additions to requirements.txt

File: setup.py (hunk 2 above)
Comment (shntnu) [reply setup.py]: I was able to install, test, and also build the docs even without adding `sphinx` and `sphinx-rtd-theme`  to `setup.py`

Comment (gwaybio) [reply setup.py]: correct. This is leftover from a previous option to autobuild. I will remove!

**Comment by gwaybio** [comment]
a26b5a9b2220102acbb282d5923e5d1e0646031a does the trick I think!

File: setup.cfg [hunk 4]
```diff
@@ -8,3 +8,7 @@ minversion =
     3.0.2
 testpaths =
     ./tests/
+
+[build_sphinx]
+project = 'cytominer_database'
+version = 0.3.3
```
Comment (shntnu) [inline setup.cfg:14]: also add this
`copyright = 2019, Allen Goodman, Claire McQuin, Shantanu Singh, Gregory Way` and remove [these](https://github.com/cytomining/cytominer-database/blob/61af4a4ce8464616de9e7c057db366beda67c70a/doc/conf.py#L51-L52) lines

Comment (gwaybio) [reply setup.cfg:14]: done in 11e49e58db5e35e05409a5b7dec3c461235acb8e

**Review by shntnu — APPROVED** [review]

**Pull request merged**